engine/table/src/main/java/io/deephaven/engine/table/impl/select/DynamicWhereFilter.java

```diff
@@ -415,32 +443,28 @@ public WritableRowSet filter(
             @NotNull final RowSet fullSet,
             @NotNull final Table table,
             final boolean usePrev) {
-        if (usePrev) {
-            throw new PreviousFilteringNotSupported();
-        }
-
         if (sourceDataIndex != null) {
             // Does our index contain every key column?
 
             if (sourceDataIndex.keyColumnNames().size() == sourceKeyColumns.length) {
                 // Even if we have an index, we may be better off with a linear search.
                 if (selection.size() > (sourceDataIndex.table().size() * 2L)) {
```

Comment:
You may want to consider the previous. size of the `sourceDataIndexTable`, (`getRowSet().sizePrev()`), if `usePrev`. L459, too.